moto/awslambda/models.py

```diff
@@ -1072,32 +1072,6 @@ def get_layer_version_by_arn(self, layer_version_arn):
         return None
 
 
-class LambdaPermission(BaseModel):
```

Comment:
This class isn't used anywhere. The functionality is already handled in `class Permission` further down in this file